### ISSUE TITLE
Daily build with latest does not work for Eclipse Job

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext#1458]
https://ci.eclipse.org/xtext/job/releng/job/nightly-build-against-latest-target/

```
node {
   echo 'Building against latest target'
   build job: 'xtext-eclipse/master', parameters: [string(name: 'target_platform', value: 'latest')], wait: false
   build job: 'xtext-xtend/master', parameters: [string(name: 'target_platform', value: 'latest')], wait: false
}
```

https://ci.eclipse.org/xtext/job/xtext-eclipse/job/master/ 
vs
https://ci.eclipse.org/xtext/job/xtext-xtend/job/master/

**Comment by cdietrich** [comment]
might be releated / have similar cause as https://github.com/eclipse/xtext/issues/1457

**Comment by cdietrich** [comment]
![Bildschirmfoto vom 2019-05-17 08-32-58](https://user-images.githubusercontent.com/238736/57907681-71aebc00-787e-11e9-8149-172e7398eb0c.png)
looks also wired on ci

**Comment by cdietrich** [comment]
vs for xtend

![Bildschirmfoto vom 2019-05-17 08-35-10](https://user-images.githubusercontent.com/238736/57907805-ba667500-787e-11e9-8966-fef621641aa6.png)

**Comment by cdietrich** [comment]
Fixed in 2.18
